### PR TITLE
docs: add additional resources

### PR DESCRIPTION
**Pull request by junr03**
Signed-off-by: Jose Nino <jnino@lyft.com>

### CODE REVIEW

File: docs/root/intro/intro.rst [hunk 1]
```diff
@@ -10,3 +10,4 @@ Introduction
   comparison
   getting_help
   version_history
+  additional_resources
```
Comment (rebello95) [inline docs/root/intro/intro.rst]: Seems like we're missing a file?

Comment (junr03) [reply docs/root/intro/intro.rst]: 🤦‍♂

File: docs/root/intro/additional_resources.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
+Additional Resources
+====================
+
+This is a collection of resources that enhance the rest of the documentation and codebase.
+
+Blog Posts
```
Comment (rebello95) [inline docs/root/intro/additional_resources.rst]: Nit: looks like the headers in our other documentation files are sentence case

File: docs/root/intro/additional_resources.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+----------
+
+- *Lyft’s Journey through Mobile Networking*
+  (`Lyft Eng Blog 01/21/2020 <https://eng.lyft.com/lyfts-journey-through-mobile-networking-d8e13c938166>`_):
+  how Lyft has evolved mobile APIs, and how that evolution lead to Envoy Mobile.
```
Comment (rebello95) [inline docs/root/intro/additional_resources.rst]:
```suggestion
  how Lyft evolved their mobile APIs, and how that evolution lead to Envoy Mobile.
```

**Comment by goaway** [comment]
nice!

**Pull request merged**